include/astshim/FitsChan.h

```diff
@@ -485,6 +487,14 @@ class FitsChan : public Channel {
     std::string getCardName() const { return getC("CardName"); }
 
     /**
+    Get the name of all cards, in order, starting from the first card
+
+    Not "const" because the code changes the index of the current card while operating
+    (but restores the original index when done).
+    */
+    std::vector<std::string> getCardNames();
```

Comment:
We can easily switch {{astshim}} names to use LSST capitalization, e.g. {{getNAxes}} instead of {{getNaxes}}. That will remove some of the pain but certainly not all.